extract_stats.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
I still think this should be in the Docker image, but I guess OK for now.

Reply:
@stffrdhrn, I will use this in https://github.com/librecores/librecores-ci-jenkins-server/issues/67 , after this PR is merged and will do changes(will send another OR ) once its library is developed , i guess that will ok ?

Reply:
We might move it right away.
Ideally I would prefer to keep it in YOSYS repo so that we do not maintain it across versions.. But it is also fine to have it in the image directly for now

Reply:
Raised https://github.com/librecores/docker-images/pull/28 for this. Will make modifications once it is merged